src/components/App.js

```diff
@@ -1,13 +1,18 @@
+import React from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Post from "./Post";
+import SignUp from "./authComponents/SignUp";
+import Login from "./authComponents/Login";
 
 
 function App() {
   return (
     <BrowserRouter>
-    <Routes>
-    <Route path="/post" element={ <Post />} />
```

Comment:
Precisar manter a rota post, para que a pagina principal funcione.

Reply:
beleza, essa rota se for a principal, tem que mudar pra /timeline

Reply:
essa rota /post não foi mudada no meu codigo não, só mudou de lugar mesmo. ela ta embaixo do signup e login

Reply:
ta no app:14